### DISCUSSION TITLE
On a un problème avec la vue Peripherique 500 - server error

### DISCUSSION THREAD

**Discussion by moiiom59** [dbarzin/mercator#523] [General]
tout est dans le titre 
merci

**Comment by dbarzin** [comment]
Avez-vous bien suivit la procédure de mise à jour de l'application ? 
Avez-vous migré la base de données ?
Quel est le message d’erreur présent dans les logs /var/www/mercator/storage/logs/laravel.log ?

**Comment by moiiom59** [comment]
L’installation date de moins de 15 jours et j’ai effectué un git pull ce jour ?
Voici le log
Merci de votre aide


Bien cordialement,

**Comment by dbarzin** [comment]
C'est normal que cela ne fonctionne pas. Il faut appliquer la procédure de mise à jour : 

https://github.com/dbarzin/mercator/blob/master/INSTALL.fr.md#mise-%C3%A0-jour

Notamment, migrer la base de données.

**Comment by moiiom59** [comment]
En effet , je pensai que le GIT s’occuper aussi de la maj des tables !!
problème résolu

Bien cordialement,
